### PR TITLE
Fix instructions to deploy EARs

### PR DESCRIPTION
**Pull request by sgilda**
This addresses the deployment failures for ejb-in-ear and kitchensink-ear. Reworked the README files as long as I was in there to better match the template.

### CODE REVIEW

File: ejb-in-ear/README.md [hunk 1]
```diff
@@ -33,68 +30,73 @@ displayed to the user.
...
 
-If you created the project using the Maven archetype wizard in your IDE
-(Eclipse, NetBeans or IntelliJ IDEA), then there is nothing to do. You should
-already have an IDE project.
+Import the project into an IDE
+---------------------
 
-Detailed instructions for using Eclipse with JBoss AS 7 are provided in the
-JBoss AS 7 <a href="https://docs.jboss.org/author/display/AS71/Getting+Started+Developing+Applications+Guide" title="Getting Started Developing Applications Guide">Getting Started Developing Applications Guide</a>.
+If you created the project using the Maven archetype wizard in your IDE (Eclipse, NetBeans or IntelliJ IDEA), there is nothing to do. You should already have an IDE project.
```
Comment (pmuir) [inline ejb-in-ear/README.md:90]: This isn't right, you can't create this app from an archetype. I've reverted locally.

**Comment by sgilda** [comment]
The deploy instructions still need to be changed though. The 2 separate commands don't work.
Sounds like I need to modify the template README also?

**Comment by pmuir** [comment]
Yes, I've just reverted the IDE bit.

**Comment by pmuir** [comment]
Merged. I fixed the indenting for code blocks to be consistent, and also added a bit of back tick action.

**Comment by sgilda** [comment]
Pete, turns out that instruction was in the README before I edited it. Because it was moved, the diff showed it as being added.
It is not in the template version so there's nothing to fix. :-)

**Pull request closed without merging**